contracts/forger_stake_v2/js/src/index.js

```diff
@@ -33,8 +33,7 @@ const setNetwork = () => {
   if (network === "eon") {
     return "https://eon-rpc.horizenlabs.io/ethv1";
   } else if (network === "gobi") {
-    // return "https://gobi-rpc.horizenlabs.io/ethv1";
-    return "https://horizen:c-ZMm4u72okCVALPA_Bb0Y@pool.zenchain.info/ethv1";
```

Comment:
Seems like basic auth creds ?) Are those temp to make sure we are not exposing them in GH history.

Reply:
@cronicc I'm sorry for the leak, can these credentials be rotated?